src/DomBuilder/DomBuilderInterface.php

```diff
@@ -36,7 +36,9 @@
 interface DomBuilderInterface
 {
     public function asXml(): string;
-    
+
+    public function getDoc(): \DomDocument;
```

Comment:
@allphat please refactor the method to `asDoc()` to unify the naming with `asXml()`